patches/chrome-installer-linux-rpm-update_package_provides.py.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/chrome/installer/linux/rpm/update_package_provides.py b/chrome/installer/linux/rpm/update_package_provides.py
+index 4ce30012357a32e1316dbb3930ce55ddf9acacfa..c8a661351b8279b5d6ce239eb4c61725c27ae8f6 100755
+--- a/chrome/installer/linux/rpm/update_package_provides.py
++++ b/chrome/installer/linux/rpm/update_package_provides.py
+@@ -32,6 +32,7 @@ PACKAGE_FILTER = [
+     "libatk-bridge-2.0.so",
+     "libc.so",
+     "libcairo.so",
++    "libcairo-gobject.so.2",
+     "libcups.so",
+     "libdbus-1.so",
+     "libdl.so",
+@@ -50,11 +51,15 @@ PACKAGE_FILTER = [
+     "libnssutil3.so",
+     "libpango-1.0.so",
+     "libpangocairo-1.0.so",
++    "libplc4.so",
++    "libplds4.so",
+     "libpthread.so",
++    "libresolv.so.2",
```

Comment:
and this?

Reply:
These are both from https://staging.ci.brave.com/job/brave-browser-build-linux/28/console
(The same is reflected in the linked issue)

Reply:
must be deps of the new deps. Ok